<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CQDlgMotionPlanning</class>
 <widget class="QDialog" name="CQDlgMotionPlanning">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>380</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Motion Planning</string>
  </property>
  <property name="locale">
   <locale language="English" country="UnitedStates"/>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>361</width>
     <height>521</height>
    </rect>
   </property>
   <property name="title">
    <string>Motion planning objects</string>
   </property>
   <property name="flat">
    <bool>true</bool>
   </property>
   <widget class="QToolButton" name="qqAddNew">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>15</y>
      <width>101</width>
      <height>18</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::StrongFocus</enum>
    </property>
    <property name="text">
     <string>Add new object</string>
    </property>
   </widget>
   <widget class="QListWidget" name="qqList">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>341</width>
      <height>111</height>
     </rect>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Illegal configuration check (phase1: self-collision)</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="qqSelfDistDist">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Entity 1</string>
     </property>
    </widget>
    <widget class="QComboBox" name="qqEntity1Combo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>251</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Entity 2</string>
     </property>
    </widget>
    <widget class="QComboBox" name="qqEntity2Combo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>251</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="qqSelfColl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>collisions</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Check for</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="qqSelfDist">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>min. distance</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>341</width>
      <height>91</height>
     </rect>
    </property>
    <property name="title">
     <string>Illegal configuration check (phase2: robot-obstacles)</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLineEdit" name="qqDistDist">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>60</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Robot</string>
     </property>
    </widget>
    <widget class="QComboBox" name="qqRobotCombo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>251</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_8">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Obstacle</string>
     </property>
    </widget>
    <widget class="QComboBox" name="qqObstacleCombo">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>251</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QRadioButton" name="qqColl">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>60</y>
       <width>71</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>collisions</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QLabel" name="label_9">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>71</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Check for</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="qqDist">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>60</y>
       <width>91</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>min. distance</string>
     </property>
     <property name="autoExclusive">
      <bool>false</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>341</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>Associated inverse kinematics</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>101</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Associated IK group</string>
     </property>
    </widget>
    <widget class="QComboBox" name="qqIkGroupCombo">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>20</y>
       <width>201</width>
       <height>18</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Base</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>60</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Tip</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>51</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Target</string>
     </property>
    </widget>
    <widget class="QLabel" name="qqBase">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>40</y>
       <width>261</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QLabel" name="qqTip">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>60</y>
       <width>261</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
    <widget class="QLabel" name="qqTarget">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>80</y>
       <width>261</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>--</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_6">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>450</y>
      <width>341</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Various</string>
    </property>
    <property name="flat">
     <bool>true</bool>
    </property>
    <widget class="QToolButton" name="qqEditJointProp">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Edit joint properties</string>
     </property>
    </widget>
    <widget class="QToolButton" name="qqEditWorkspaceMetric">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>20</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Edit workspace metric</string>
     </property>
    </widget>
    <widget class="QToolButton" name="qqComputeNodes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>151</width>
       <height>18</height>
      </rect>
     </property>
     <property name="focusPolicy">
      <enum>Qt::StrongFocus</enum>
     </property>
     <property name="text">
      <string>Compute phase1 nodes</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="showPhase1Nodes">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>40</y>
       <width>151</width>
       <height>17</height>
      </rect>
     </property>
     <property name="text">
      <string>Visualize phase1 nodes</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>qqAddNew</tabstop>
  <tabstop>qqList</tabstop>
  <tabstop>qqIkGroupCombo</tabstop>
  <tabstop>qqEntity1Combo</tabstop>
  <tabstop>qqEntity2Combo</tabstop>
  <tabstop>qqSelfColl</tabstop>
  <tabstop>qqSelfDist</tabstop>
  <tabstop>qqSelfDistDist</tabstop>
  <tabstop>qqRobotCombo</tabstop>
  <tabstop>qqObstacleCombo</tabstop>
  <tabstop>qqColl</tabstop>
  <tabstop>qqDist</tabstop>
  <tabstop>qqDistDist</tabstop>
  <tabstop>qqEditJointProp</tabstop>
  <tabstop>qqEditWorkspaceMetric</tabstop>
  <tabstop>qqComputeNodes</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
